_posts/2022-07-27-#24858.md

```diff
@@ -10,15 +10,88 @@ status: upcoming
 commit: bcb0cac
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- A new node automatically initializes and populates its data directory by performing
+["Initial block download" (IBD)](https://bitcoin.stackexchange.com/a/106136/97099),
+which is the process of fetching blocks
+from peers and storing them in the `blocks` directory within the data directory
+(default `$HOME/.bitcoin`). 
+
+- The blocks are stored in files named `blknnnnn.dat` (for example, `blk01234.dat`).
+These files are limited to 128 MiB, so each can hold several hundred blocks.
+
+- The `blknnnnn.dat` files are not in any standard database format; their format
+is quite simple: a list of blocks (their serializations), each preceded by
+a "marker" or "magic number" to indicate the start of a block, and the block's
+size or length in bytes. The blocks need not be in height-order, either within a
```

Comment:
I would also mention the block header which comes between the magic and the actual blockdata, since it's important for the PR.

Reply:
I don't understand, it's the 4-bytes magic, the 4-byte block size (length) -- maybe I should mention those 4-byte sizes to make it concrete -- followed by the serialization of the block (the block header followed by the transactions). The block header isn't serialized separately. Am I missing something?

Reply:
Oh, you are right, the block header vs serialized transaction part is not important in this context. You can disregard this comment.